examples/apps/collaborative-textarea/webpack.config.js

```diff
@@ -5,7 +5,7 @@
 
 const fluidRoute = require("@fluid-tools/webpack-fluid-loader");
 const path = require("path");
-const merge = require("webpack-merge");
+const { merge } = require("webpack-merge");
```

Comment:
It looks like this is only used to merge the dev/prod setting
@tylerbutler Do you know if our examples are ever used in a way that broadly requires that dev/prod split?  If not it seems like that may be worth cleaning up for the examples.